playwright/_impl/_browser_type.py

```diff
@@ -209,7 +212,14 @@ async def connect(
         browser._is_remote = True
         browser._is_connected_over_websocket = True
 
-        transport.once("close", browser._on_close)
+        def handle_transport_close() -> None:
```

Comment:
Why is this a part of the roll? Is it a bug fix?

Reply:
The disconnect handling was changed from you in the last roll, I adopted your changes.